index.rst

```diff
@@ -3,6 +3,7 @@ Welcome to PyTorch Tutorials
 
 What's new in PyTorch tutorials?
 
+* `Survey: Let us know how the PyTorch Tutorials are working for you! <https://docs.google.com/forms/d/e/1FAIpQLSfM-n7fF-K8AqkOcdSlzS_0g-xd6ith7U_W36xG-C_6Q2B6Mg/viewform>`__
```

Comment:
looks good, but maybe we could make it a ..note:: so it's more discoverable. What do you think?